src/View/View.php

```diff
@@ -78,7 +78,7 @@ class View implements EventDispatcherInterface
     }
     use EventDispatcherTrait;
     use InstanceConfigTrait {
-        getConfig as private _getConfig;
+        getConfig as protected;
```

Comment:
@ADmad Was this an attempt to hide the function?

Reply:
This was done in 4.x because `getConfig()` needed to be overriden to show warnings for usage of deprecated special view vars. This can be totally removed now.

Reply:
So change to just `use InstanceConfigTrait;` now.